battleship.py

```diff
@@ -194,7 +219,14 @@ def shipIsValid(grid, ship):
 Returns: None
 '''
 def placeShip(data):
-    return
+    user_board = data["usergrid"]
+    if shipIsValid(user_board,data["tempship"]):
+        for i in data["tempship"]:
+           user_board[i[0]][i[1]]= SHIP_UNCLICKED
+    else:
+      print("Ship is not valid") 
+    data["tempship"]=[]   

```

Comment:
Count the number of ships which have been placed

Reply:
Also, resolve conflicts by deleting pycache and then clicking on the resolve conflicts button for battleship.py

Reply:
issue resolved